daemon/src/order/maker.rs

```diff
@@ -54,7 +52,7 @@ pub struct Actor {
     n_payouts: usize,
     decision_senders: HashMap<OrderId, oneshot::Sender<protocol::Decision>>,
     db: sqlite_db::Connection,
-    latest_offers: MessageChannel<xtra_libp2p_offer::maker::GetLatestOffers, Option<MakerOffers>>,
+    latest_offers: MessageChannel<xtra_libp2p_offer::maker::GetLatestOffers, Vec<model::Offer>>,
```

Comment:
❓Why do we use a `Vec` here and not a `Hashmap` that indexes per `symbol` as well? Or hide the whole abstraction behind a struct that hides how this is implemented?

Reply:
I wanted to keep the offer actor as simple as possible. I didn't see the need to use a hasmap because a simple list works, but we can consider it if there is a good reason for it.

Reply:
Taking as feedback to wrap Vec<model::Offer> around a offer protocol newtype.

Reply:
Yeah, that way we can make the `impl` of `model::Offers` explicit to the combinations that can be set at the moment and the internal representation does not matter.

For the `Offers` sent over the wire we can consider a model that uses a `Hashmap` - or our `model::Offers` can ensure that the `protocol::Offers` is sane when converting into it. We can opt to keep a `Vec` internally for both, but then then the accessors should ensure that one can only set sane combinations and no duplicates.